sdf.md

```diff
@@ -197,59 +226,127 @@ For example, an Object definition looks like this:
   }
 }
 ~~~
+{: #exobject title="Example Object definition"}
 
-An Object "foo" is defined in the default namespace, containing a property "foo.bar",  of type boolean.
+This example defines an Object "foo" is defined in the default namespace, containing a property "foo.bar",  of type boolean.
+<!-- XXX We are not using the foo.bar style, no? -->
```

Comment:
I guess we currently call that simply "odmProperty bar"

Reply:
This is why dot notation is problematic, but I think we could live with name munging as is done in Python. For formal references we should just stick to JSON pointer - odm:/odmObject/foo/odmProperty/bar